resources/lib/ipwww_radio.py

```diff
@@ -578,26 +517,16 @@ def ParseStreams(stream_id):
     retlist = []
     # print "Parsing streams for PID: %s"%stream_id[0]
     # Open the page with the actual strem information and display the various available streams.
-    NEW_URL = "http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/iptv-all/vpid/%s" % stream_id[0]
+    NEW_URL = "http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/apple-ipad-hls/vpid/%s/proto/http?cb=%d" % (stream_id[0], random.randrange(10000,99999)) #NOTE magic from get_iplayer
```

Comment:
I don't know why but it works. Some of the servers have cp prefixes like http://cp401492-vh.akamaihd.net. Maybe it is random queue preference for load balancing. I haven't got an iThing to dig deeper. It was the only international stream type I could find that would work in Kodi. I'm sure the international users would be greatful for something rather than nothing.